jdaviz/configs/cubeviz/helper.py

```diff
@@ -168,6 +168,24 @@ def get_data(self, data_label=None, spatial_subset=None, spectral_subset=None, f
                               spectral_subset=spectral_subset, function=function,
                               cls=cls, use_display_units=use_display_units)
 
+    # Need these methods for Imviz Aperture Photometry plugin.
+
+    def get_link_type(self, data_label_1, data_label_2):
```

Comment:
Note to self: See if we still need this after rebasing on `main` now that #2179 is merged, just in case.